src/main/java/gtPlusPlus/core/recipe/RecipesSeleniumProcessing.java

```diff
@@ -30,33 +27,31 @@ public static void init() {
 
         // Liquify the Dried Dioxide
         GTValues.RA.stdBuilder()
-            .itemInputs(GTUtility.getIntegratedCircuit(13), MaterialMisc.SELENIUM_DIOXIDE.getDust(1))
+            .itemInputs(MaterialMisc.SELENIUM_DIOXIDE.getDust(1))
             .fluidInputs(FluidUtils.getSteam(500))
             .fluidOutputs(MaterialMisc.SELENIUM_DIOXIDE.getFluidStack(1000))
             .duration(24 * SECONDS)
             .eut(1024)
-            .addTo(pyrolyseRecipes);
+            .addTo(cokeOvenRecipes);
 
         // Produce Selenious Acid
         GTValues.RA.stdBuilder()
-            .itemInputs(GTUtility.getIntegratedCircuit(14), MaterialMisc.SELENIUM_DIOXIDE.getCell(1))
-            .itemOutputs(CI.emptyCells(1))
-            .fluidInputs(FluidUtils.getHotWater(4000))
+            .fluidInputs(FluidUtils.getHotWater(4000), MaterialMisc.SELENIUM_DIOXIDE.getFluidStack(1000))
             .fluidOutputs(MaterialMisc.SELENIOUS_ACID.getFluidStack(1000))
             .duration(24 * SECONDS)
+            .noOptimize()
             .eut(TierEU.RECIPE_EV)
-            .addTo(pyrolyseRecipes);
+            .addTo(cokeOvenRecipes);
 
         // Make Selenium
         GTValues.RA.stdBuilder()
             .itemInputs(GTUtility.getIntegratedCircuit(14), MaterialsElements.getInstance().CARBON.getDust(16))
-            .itemOutputs(
-                MaterialsElements.getInstance().SELENIUM.getIngot(1),
-                MaterialsElements.getInstance().SELENIUM.getIngot(1))
-            .outputChances(100_00, 20_00)
             .fluidInputs(MaterialMisc.SELENIOUS_ACID.getFluidStack(750), Materials.SulfuricAcid.getFluid(8000))
-            .fluidOutputs(MaterialsElements.getInstance().SELENIUM.getFluidStack(144 * 1))
+            .fluidOutputs(
+                MaterialsElements.getInstance().SELENIUM.getFluidStack(324),
```

Comment:
```suggestion
                MaterialsElements.getInstance().SELENIUM.getFluidStack(144 * 2 + 36),
```
Just for readability